frontend/src/components/Layout.tsx

```diff
@@ -4,18 +4,21 @@ import { useTranslation } from 'next-i18next'
 
 import Footer from './Footer'
 import Header from './Header'
+import { BreadcrumbItemType } from './Breadcrumb'
 
 export interface LayoutProps {
   children: ReactNode
+  breadcrumbItems: BreadcrumbItemType[];
```

Comment:
Make it optional (undefined).

```suggestion
  breadcrumbItems?: BreadcrumbItem[];
```

Reply:
Fixed